### PR TITLE
Portfolio base

### PR DESCRIPTION
**Pull request by FromATP**
In portfolio_base, I create a website in imitation of _examples_
which includes _index.html, gallery.html_ and _aboutme.html_.
I put a short introduction on me in _index.html_, my album in _gallery.html_ and my personal experiences in _aboutme.html_
It's a very simple website. I will improve it later.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,42 @@
...
+        <p>Click <a href="gallery.html">here</a> to see more!</p>
+
+      <h2>Let's make friends!</h2>
+        <p>E-mail: zhangyx00@pku.edu.cn</p>
+        <p>Wechat: FromATP</p>
+        <p>Steam Friend Code: 919051252 (XD </p>
+
+<!--    <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>-->
```
Comment (ivancjw) [inline portfolio/src/main/webapp/index.html:40]: Pls remove

Comment (FromATP) [reply portfolio/src/main/webapp/index.html:40]: tmp

I have deployed it on Google Cloud Platform, on http://yzhang2-sps-summer20.appspot.com/

Comment (FromATP) [reply portfolio/src/main/webapp/index.html]: Redundant comments have been removed.

Comment (ivancjw) [reply portfolio/src/main/webapp/index.html]: You can mark the conversation as resolved once you believe no more action is needed from you side.

**Pull request merged**